torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -2330,6 +2334,10 @@ struct to_ir {
         return "__xor__";
       case TK_IN:
         return "__contains__";
+      case TK_LSHIFT:
+        return "<<";
```

Comment:
These correspond to the operator -> python magic method name. so `TK_SHIFT` should be `__lshift__` so that we property desugar `<<` when invoked with a class that defines `__lshift__`. Probably `getOperatorOverload` can be renamed to `getPythonMagicMethodName`